parser-typechecker/unison-parser-typechecker.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 
--- This file has been generated from package.yaml by hpack version 0.35.0.
+-- This file has been generated from package.yaml by hpack version 0.34.4.
```

Comment:
We should standardize all of our tooling so these diffs go away 😓. Use nix-provided `stack`, `hpack`, `ghc`,  etc., probably?

Reply:
How do we do that?

Reply:
and then update development.markdown

Reply:
Also I'm surprised an older version overwrote a newer version, was that intentional?

Reply:
Yes, because otherwise the cabal file doesn't get generated. I had to delete it manually and rebuild to regenerate